content/customize-dock.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Dock (Move or Customize)
+description: >
+  How to move the Dock to different areas of the screen and customize other settings.
+keywords:
+  - Launcher
+  - Bottom
+  - Left
+  - Moving
+  - Dock
+  - Dash to Dock
+  - Pop Dock
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+
+redirect_from:
+  - /movelauncher
+  - /articles/move-launcher
+---
+
+Pop!\_OS and Ubuntu both contain an application dock on the bottom of the screen. You can adjust this preference, move the dock to the bottom of the screen, and more in <u>Settings</u>.
+
+## Default Dock on Pop!\_OS
+
+Pop!\_OS includes a lightweight version of Dash to Dock, which is why the dock already displays on the bottom of the screen by default.
+
+![Stock Pop Dock](/images/customize-dock/pop-dock.png)
+
+## Default Dock on Ubuntu
+
+Ubuntu includes a lightweight version of Dash to Dock, which is why the dock already displays on the left side of the screen by default.
```

Comment:
I think Dash to Dock sets the location to the bottom edge by default, so saying that's why it's on the left edge seems confusing.